src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,52 @@
+package duke.task;
+
+/**
+ * Abstract Task class.
+ */
+public abstract class Task {
```

Comment:
Interesting use of abstract class, didnt think of making my Task class abstract haha